bin/cloud-push

```diff
@@ -31,6 +31,6 @@ done
 
 echo "Deploy this build to staging by running:"
 echo
-echo "    $ bin/mz --profile=staging login --endpoint=https://staging.cloud.materialize.com"
+echo "    $ bin/mz --profile=staging login --endpoint=https://staging.console.materialize.com"
```

Comment:
I think `mz` is a bit broken with interactive logins right now, but I think the existing `staging.cloud.materialize.com` endpoint is closer to correct than `staging.console.materialize.com`.

Reply:
@joacoc is it right that this should still be `cloud`?

Reply:
I think applying a quick tweak would be worth it to unblock developers. But I think this should stay as `admin.staging.cloud.materialize.com`, and then the tweak in `mz` should rewrite "cloud" to "console" when doing a browser login?

Reply:
Make sense! Will prepare the PR.

Reply:
Thank you!

Reply:
@antiguru if you back out this part of the change, I think this is good to merge?

Reply:
Yup, removed.